pkg/cmd/cli/kubeflowmodelregistry/kfmr.go

```diff
@@ -337,13 +338,38 @@ func (m *ModelServerPopulator) GetAuthentication() *bool {
 	return &auth
 }
 
+// GetName returns the inference server name, sanitized to meet the following criteria
+// "a string that is sequences of [a-zA-Z0-9] separated by any of [-_.], at most 63 characters in total"
 func (m *ModelServerPopulator) GetName() string {
 	if len(m.InferenceServices) > m.InfSvcIndex {
-		return m.InferenceServices[m.InfSvcIndex].GetName()
+		sanitizedName := sanitizeName(m.InferenceServices[m.InfSvcIndex].GetName())
+		return sanitizedName
 	}
 	return ""
 }
 
+func sanitizeName(name string) string {
+	sanitizedName := name
+
+	// Replace any invalid characters with an empty character
+	validChars := regexp.MustCompile(`[^a-zA-Z0-9\-_.]`)
```

Comment:
maybe add a `nameRegexp` constant and define it up top with `tagRegexp` in this file

Reply:
tell you what I can do that when I rebase